exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,21 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+    return num < 0 ? true : false;
```

Comment:
It's a nice shortcut, is that mean the same as 
return true and then return false?

Reply:
Yes, this is a ternary operator, it's a short way of writing an if statement. You can find out about it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)

Reply:
You've done this below but just as a reminder you can actually just return the boolean part of this directly like this without the ternary:
```suggestion
    return num < 0;
```

The reason for this is because `num < 0` actually is already equal to `true` or `false`. With the ternary it is like saying: `if (num < 0 === true) { return true } ....`